docs/design/autoware-architecture/index.md

```diff
@@ -0,0 +1,43 @@
+# Architecture overview
+
+This page describes the architecture of Autoware.
+
+## Introduction
+
+The current Autoware is defined to be a layered architecture that clarifies each module's role and simplifies the interface between them. By doing so:
+
+- Autoware's internal processing becomes more transparent.
+- Collaborative development is made easier because of the reduced interdependency between modules.
+- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface.
+
+Note that the initial focus of this architecture design was solely on driving capability, and so the following features were left as future work:
+
+- Fail safe
+- HMI
```

Comment:
I think this is the first time people will come across this term, so we should expand it rather than just have an acronym.

```suggestion
- Human Machine Interface
```

Reply:
Thank you. Addressed.